README.md

```diff
@@ -7,7 +7,7 @@ Lockbox for Firefox is a work-in-progress extension for Firefox to improve upon
 Firefox's built-in password management. If you're interested, you should
 probably come back later when we have more to show!
 
-## Quick Start
+## [Quick Installation](install-link)
```

Comment:
If you're using a footnoted link like this, you need to use square brackets instead of parens.

Reply:
>you need to use square brackets

Oh shoot, good catch. Thank you...